src/compiler/types.ts

```diff
@@ -6574,8 +6574,8 @@ namespace ts {
         exactOptionalPropertyTypes?: boolean;
         experimentalDecorators?: boolean;
         forceConsistentCasingInFileNames?: boolean;
-        /*@internal*/generateCpuProfile?: string;
-        /*@internal*/generateTrace?: string;
+        generateCpuProfile?: string;
+        generateTrace?: string;

```

Comment:
I  think these should be internal only. Infact one of them is marked command line only .. we should mark the other one too but definitely dont see reason to expose these